projects/storefrontlib/src/cms-components/product/product-list/product-grid-item/product-grid-item.component.html

```diff
@@ -32,3 +32,15 @@
   [showQuantity]="false"
   [productCode]="product.code"
 ></cx-add-to-cart>
+<a
+  *ngIf="product.configurable"
+  [routerLink]="
+    {
+      cxRoute: 'configure' + product.configuratorType,
+      params: { rootProduct: product.code }
+    } | cxUrl
+  "
+  class="btn btn-primary btn-block btn-to-configuration"
+  cxAutoFocus
+  >{{ 'header.toconfig' | cxTranslate }}</a
```

Comment:
I'm I spot it lately now.

The translation chunks are created to lazy load the translations only needed for a specific feature.
To know which master key belongs to which chunk, we introduced translation chunks mapping (`translation-chunks.config.ts`).

The master translation keys  `header`  and `attribute` are too general to assign them only to the (lazy-loaded) translation chunk dedicated for the configurator feature.
Can we scope the keys needed for the configurator features in a one master key `configurator`, please?

Reply:
I did this change in feature/GH-4358.
And just to be sure: In translation-chunks.config.ts this looks like this then, right?

configurator: ['configurator.header', 'configurator.attribute'],